spring-cloud-task-docs/src/main/asciidoc/repository.adoc

```diff
@@ -0,0 +1,32 @@
+[[task-repository]]
+= Spring Cloud Task Repository
+
+[[partintro]]
+--
+Spring Cloud Task uses a relational database to store the results of an
```

Comment:
"Out of the box..."  There isn't really any reason to not have non-relational repository implementations for tasks so we should be clear that this is just what we provide.  We also cover some of this within the getting-started so we should consolidate/reference/etc.